pkg/occlient/occlient.go

```diff
@@ -2629,12 +2606,22 @@ func makeTar(srcPath, destPath string, writer io.Writer, files []string) error {
 	srcPath = path.Clean(srcPath)
 	destPath = path.Clean(destPath)
 
+	glog.V(4).Infof("makeTar: srcPath: %s, destPath: %s, files: %+v", srcPath, destPath, files)
```

Comment:
can we have a little bit more meaningful message here? 

If this is meant to show with what arguments `makeTar` was called than maybe something like `makeTar arguments:` or something similar that desribes what this message is showing.

Reply:
Ack